lib/Ama.pm

```diff
@@ -96,6 +102,7 @@ $self->plugin("OAuth2Accounts" => {
   $r->delete('/questions/:question_id')->to('questions#remove')->name('remove_question'); # Delete from DB and redirect to questions
   $r->delete('/removeAll')->to('questions#removeAll')->name('removeAll'); #Delete every question
```

Comment:
This is something we talked about earlier. I assume Stefan found it again while doing this code review and brought it up again. I'll go ahead and create an issue